DockDoor/Utilities/DockObserver.swift

```diff
@@ -539,4 +549,20 @@ final class DockObserver {
             }
         }
     }
+
+    private func handleCmdRightClickQuit(app: NSRunningApplication, event: CGEvent) {
+        Task { @MainActor in
+            // Check if Option key is also pressed for force quit
+            let shouldForceQuit = event.flags.contains(.maskAlternate)
+
+            if shouldForceQuit {
```

Comment:
Since shouldForceQuit is initialized but not used anywhere else, can we move the `event.flags.contains(.maskAlternate)` call to be inline with the `if` statement?